components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -257,9 +257,10 @@ - (CGSize)intrinsicContentSize {
 }
 
 - (CGSize)sizeThatFits:(CGSize)size {
+  self.leadingUnderlineLabel.preferredMaxLayoutWidth = size.width;
+  self.trailingUnderlineLabel.preferredMaxLayoutWidth = size.width;
```

Comment:
`sizeThatFits:` generally shouldn't have side effects; at minimum can we modify and then restore these properties?